netlify-templates/victor-hugo
# Target on arm64 (apple m1)

This pull request target to be used on dev environment for macOS BigSur on arm64 (Apple Silicon)

- Bump node to latest LTS
- Bump hugo-bin@latest for arm64
- Upgrade to webpack 5
- Remove deprecated dev packages
- Install new dev packages

Should work if you do :

use node v14.14.1 use `nvm` or `fnm`

`npm install`

`npm start`

`npm build`

You should see only warning and no error on the terminal.
